package.json

```diff
@@ -34,7 +34,7 @@
 		"workers-prod": "SEQUELIZE_MAX_CONNECTIONS=1 NODE_PATH=./dist/server/client:./dist/server WORKER=true node dist/server/workers/init"
 	},
 	"dependencies": {
-		"@benrbray/prosemirror-math": "0.2.2",
+		"@benrbray/prosemirror-math": "^0.2.2",
```

Comment:
Let's revert this change, it's important that we keep this package pegged to a specific version because we're fucking around with its internals so much 😅